GG/src/RichText/TextBlock.h

```diff
@@ -0,0 +1,31 @@
+#ifndef TEXTBLOCK_H
+#define TEXTBLOCK_H
+
+#include <GG/TextControl.h>
+#include <GG/RichText/BlockControl.h>
+
+namespace GG
+{
+
+/**
+ * @brief A text control embeddable in RichText.
+*/
+class TextBlock : public GG::BlockControl
+{
+public:
+    //! Create a new TextBlock.
+  TextBlock(X x, Y y, X w, const std::string& str, const boost::shared_ptr<Font>& font, Clr color,
```

Comment:
no two space indents. the public: above is good, though
also, make sure the comment left-side lines up with the following line